3135/3/app.rb

```diff
@@ -0,0 +1,77 @@
+require 'sinatra'
+require 'sinatra/activerecord'
+require 'sinatra/flash'
+require 'pry'
+
+set :database, 'sqlite3:db/database.sqlite3'
+
+enable :sessions
+
+before do
+  @active_user = User.find_by(id: session[:id])
+
+  pass if ['/login', '/logout', '/registration', '/add_review'].include? request.path_info
```

Comment:
That's a bad idea. So there will be a moment when you forget about adding some new URL to this filter

Reply:
https://github.com/gnab/sinatra-authorize here are some thoughts on authorization